ueye_cam:
  ros__parameters:
    # For default values, refer to:
    #  - ueye_cam/camera_parameters.hpp
    #  - ueye_cam/node_parameters.hpp
    camera_name: froody
    frame_name: froody
    camera_id: 0                # 0 : ANY_CAMERA
    color_mode: rgb8
    auto_gain: true
    auto_white_balance: true